resources/js/components/Leaderboard.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="w-full relative bg-white text-black mt-16">
+    <table class="w-full mx-auto text-left">
+      <thead class="rankone font-extrabold text-white">
+        <tr class="h-12">
+          <td class="pl-2">Rank</td>
+          <td class="text-center">
+            <span>
+              <img
+                :src="sortedUsers[0].avatar_url"
+                class="w-16 h-16 border-white border-2 shadow-lg rounded-full mx-auto left-0 right-0 absolute"
+              />
+            </span>
+          </td>
+          <td class="text-right pr-2">Score</td>
+        </tr>
+      </thead>
+      <tbody>
+        <tr class="rankone font-extrabold text-2xl text-white">
+          <td class="pl-2">1</td>
+          <td class="overflow-hidden text-center">{{sortedUsers[0].name}}</td>
+          <td class="text-right pr-2">{{sortedUsers[0].score}}</td>
+        </tr>
+
+        <tr class="table-row" v-for="(user, key) in sortedUsers" v-if="key != 0">
+          <td class="pl-2 py-2 font-bold">{{++key}}</td>
+          <td class="py-2">
+            <span>
+              <img :src="user.avatar_url" class="rounded-full h-8 w-8 inline-block shadow-2xl"/>
+            </span>
+            {{ user.name }}
+          </td>
+          <td class="pr-2 py-2 text-right font-bold">{{user.score}}</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+export default {
```

Comment:
I don't think you even need vue, it is not a real-time leaderboard. Please use blade for now.

Reply:
I assumed it to be real-time.